docs/reference/dependencies.md

```diff
@@ -3156,1011 +3156,8 @@ All of these libraries are required for core functionality.
-- [symfony/console@v7.1.3](https://github.com/symfony/console.git) (MIT)
-- [symfony/css-selector@v7.1.1](https://github.com/symfony/css-selector.git) (MIT)
-- [symfony/deprecation-contracts@v3.5.0](https://github.com/symfony/deprecation-contracts.git) (MIT)
-- [symfony/error-handler@v7.1.3](https://github.com/symfony/error-handler.git) (MIT)
-- [symfony/event-dispatcher@v7.1.1](https://github.com/symfony/event-dispatcher.git) (MIT)
-- [symfony/event-dispatcher-contracts@v3.5.0](https://github.com/symfony/event-dispatcher-contracts.git) (MIT)
-- [symfony/finder@v7.1.3](https://github.com/symfony/finder.git) (MIT)
-- [symfony/http-foundation@v7.1.3](https://github.com/symfony/http-foundation.git) (MIT)
-- [symfony/http-kernel@v7.1.3](https://github.com/symfony/http-kernel.git) (MIT)
-- [symfony/mailer@v7.1.2](https://github.com/symfony/mailer.git) (MIT)
-- [symfony/mime@v7.1.2](https://github.com/symfony/mime.git) (MIT)
-- [symfony/polyfill-ctype@v1.30.0](https://github.com/symfony/polyfill-ctype.git) (MIT)
-- [symfony/polyfill-intl-grapheme@v1.30.0](https://github.com/symfony/polyfill-intl-grapheme.git) (MIT)
-- [symfony/polyfill-intl-idn@v1.30.0](https://github.com/symfony/polyfill-intl-idn.git) (MIT)
-- [symfony/polyfill-intl-normalizer@v1.30.0](https://github.com/symfony/polyfill-intl-normalizer.git) (MIT)
-- [symfony/polyfill-mbstring@v1.30.0](https://github.com/symfony/polyfill-mbstring.git) (MIT)
-- [symfony/polyfill-php72@v1.30.0](https://github.com/symfony/polyfill-php72.git) (MIT)
-- [symfony/polyfill-php80@v1.30.0](https://github.com/symfony/polyfill-php80.git) (MIT)
-- [symfony/polyfill-php83@v1.30.0](https://github.com/symfony/polyfill-php83.git) (MIT)
-- [symfony/polyfill-uuid@v1.30.0](https://github.com/symfony/polyfill-uuid.git) (MIT)
-- [symfony/process@v7.1.3](https://github.com/symfony/process.git) (MIT)
-- [symfony/routing@v7.1.3](https://github.com/symfony/routing.git) (MIT)
-- [symfony/service-contracts@v3.5.0](https://github.com/symfony/service-contracts.git) (MIT)
-- [symfony/string@v7.1.3](https://github.com/symfony/string.git) (MIT)
-- [symfony/translation@v7.1.3](https://github.com/symfony/translation.git) (MIT)
-- [symfony/translation-contracts@v3.5.0](https://github.com/symfony/translation-contracts.git) (MIT)
-- [symfony/uid@v7.1.1](https://github.com/symfony/uid.git) (MIT)
-- [symfony/var-dumper@v7.1.3](https://github.com/symfony/var-dumper.git) (MIT)
-- [tijsverkoyen/css-to-inline-styles@v2.2.7](https://github.com/tijsverkoyen/CssToInlineStyles.git) (BSD-3-Clause)
-- [vlucas/phpdotenv@v5.6.1](https://github.com/vlucas/phpdotenv.git) (BSD-3-Clause)
-- [voku/portable-ascii@2.0.1](https://github.com/voku/portable-ascii.git) (MIT)
-- [webmozart/assert@1.11.0](https://github.com/webmozarts/assert.git) (MIT)
+- **Dependencies:** SBOM CycloneDX files with up-to-date lists of third party libraries used and their licenses can be requested on demand.
+- **Source code:** Source code access can be provided on demand.
```

Comment:
```suggestion
- **Source code:** Access to source code access is provided on [on-demand](dependency-request@camunda.com).
```

Reply:
✅ Added an email address link to both bullet points.